.github/scripts/encrypt_credentials.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Do we ever need to be able to encrypt credentials from a local dev env? If not, I think it'd be fine to not include this file

No strong opinions here, though.

Reply:
This might only be needed if we're doing credential rotation, but it's not critical for now, so I deleted it.